Refactor of Tagging Section (#7516)
DataDog/documentation: content/en/logs/log_collection/go.md

}
```

> ## Connect your service across Logs and APM

Author: ```suggestion
## Connect your service across logs and traces
```

}
```

> ## Connect your service across Logs and APM

Reviewer: I believe we should keep the word `traces`.

We are not saying Log Management here as we are talking about the data points, not the products. So we are connecting the traces and the logs, not Log Management and APM.

Author: Valid point, will update the headers

> If APM is enabled for this application, the APM tracer can help connect your logs with APM data.
> See [APM Go logging instructions][3] to learn more about adding trace IDs, span IDs, and even the
> `env`, `service`, and `version` to your logs.

Reviewer: ```suggestion
If APM is enabled for this application, connect your logs and traces by automatically adding trace IDs, span IDs,
`env`, `service`, and `version` to your logs by [following the APM Go instructions][3]
```

> **Note**: If the APM tracer injects `service` into your logs, there is no need to set a `service` field in your logs configuration.

Reviewer: ```suggestion
**Note**: If the APM tracer injects `service` into your logs, it overrides the value set in the agent configuration.
```
